float32[] joint_target
bool constrained
geometry_msgs/Quaternion target
float32[] tolerances
---
bool success
moveit_msgs/RobotTrajectory plan